src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -9,13 +9,624 @@
     }) }}
 </div>
 
-{{ include('@WebProfiler/Profiler/base_js.html.twig') }}
-
 <style{% if csp_style_nonce %} nonce="{{ csp_style_nonce }}"{% endif %}>
     {{ include('@WebProfiler/Profiler/toolbar.css.twig') }}
 </style>
-<script{% if csp_script_nonce %} nonce="{{ csp_script_nonce }}"{% endif %}>/*<![CDATA[*/
-    (function () {
-        Sfjs.loadToolbar('{{ token }}');
-    })();
-/*]]>*/</script>
+
+<script>
+    window.addEventListener('DOMContentLoaded', () => {
+        new SymfonyWebDebugToolbar('{{ token }}');
```

Comment:
How does this behaves with Turbo ?

Previously, we would have a `loadToolbar` call executed for each Turbo visits (as it would execute the new script tag). Now, we have it only on `DOMContentLoaded` whcih won't be triggered again during a Turbo visit.